features/components/index.tsx

```
@@ -3,60 +3,70 @@ import { createElement, useEffect, useState, type JSX } from "react";
         elementname: elementName,
         props: encodeURI(JSON.stringify(props)),
       });
 
     if (id) {
-      const El = globalThis?.__BUNEXT_dynamicComponents__?.find(
+      const El = globalThis?.__BUNEXT_DYNAMIC_COMPONENTS__?.find(
         (p) => p.id == id
       );
       if (!El) return fallback;
       return createElement(El.element.type, {
         dangerouslySetInnerHTML: {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)